src/pages/search/advanced_search/FundingSourceFilter.tsx

```diff
@@ -0,0 +1,69 @@
+import { Autocomplete } from '@mui/material';
+import { useQuery } from '@tanstack/react-query';
+import { useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import { fetchFundingSources } from '../../../api/cristinApi';
+import { ResultParam } from '../../../api/searchApi';
+import { AutocompleteTextField } from '../../../components/AutocompleteTextField';
+import { setNotification } from '../../../redux/notificationSlice';
+import { FundingSource } from '../../../types/project.types';
+import { getLanguageString } from '../../../utils/translation-helpers';
+
+export const FundingSourceFilter = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const paramName = ResultParam.FundingSource;
+  const history = useHistory();
+  const searchParams = new URLSearchParams(history.location.search);
+  const currentSearchTerm = searchParams.get(paramName) || '';
```

Comment:
ser ikke ut som å defaulte til tom streng i stedet for null trengs her? 🤔 
```suggestion
  const currentSearchTerm = searchParams.get(paramName);
```